pkg/cluster/internal/create/nodes.go

```diff
@@ -154,6 +162,7 @@ func nodesToCreate(cfg *config.Cluster, clusterName string) []nodeSpec {
 		}
 	}
 	isHA := controlPlanes > 1
+	ipv6 := "ipv6" == cfg.Networking.IPFamily
```

Comment:
Job is failing because of this

Reply:
Which one? Is this staticcheck? This code looks legitimate to me

Reply:
It was because it doesn´t allow Yoda conditions 👽  :) I´ve alredy fixed it